jupyter_notebook/analytics.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime, timedelta
```

Comment:
In time, yes - I want to discuss what's going on before polish the code and add things like tests and docs.